### DISCUSSION TITLE
[4.0] Feature request: Make 'save as copy' available in front end

### DISCUSSION THREAD

**Discussion by Scrabble96** [joomla/joomla-cms#28092] [2. Cold Features]
### Is your feature request related to a problem? Please describe.
For non-techy editors who don't know much html or css, it is difficult to start some articles from scratch, especially if they'd like to keep it more or less the same format as existing articles. 

At the moment, the only way to do this is to: 
1. open an existing article, 
2. copy the code, 
3. close the article, 
4. start a new one, 
5. paste in the code and then 
6. Edit

### Describe the solution you'd like
Add the 'Save as Copy' option to the front editor so, 
1. Open article
2. Click 'save as copy'
3. Edit

**Comment by Scrabble96** [comment]
Hello. It's a year since I suggested this and got three thumbs up for it, but no comments have been made to-date.

**Reply by richard67** [reply]
It seems not to be a bad idea. Maybe @nibra can have a look on it if it is worth to go into the official RFC process and if yes, advise on what to do for it?

**Reply by nibra** [reply]
We could start the RFC process for that, but I think, it is too obvious that this should be added.
Maybe we should think bigger here and start an RFC for "Simplified Administration" where this could be one point?

**Reply by richard67** [reply]
Well it's about frontend editing, so not sure if the term "administration" fits to it.

**Reply by nibra** [reply]
It could be called "Simplified Content Creation" as well. I don't have any strong oppinion about that.

**Reply by brianteeman** [reply]
Does a super simple feature request really need to go through the bureaucracy of the rfc process.

**Reply by richard67** [reply]
Not sure. That's why I had asked.

**Reply by brianteeman** [reply]
It would kill development

(I agree with major stuff)

**Reply by nibra** [reply]
> Does a super simple feature request really need to go through the bureaucracy of the rfc process.

No, that's why I said it should be part of a bigger thing, if it should go through the RFC process. In my oppinion, it should just be implemented, I personally see nothing to discuss about.

**Reply by richard67** [reply]
Would that be for 4.1 then?

**Reply by richard67** [reply]
I worry it could be forgotten here in discussions or in the many issues we have when we replace it by an issue. Having it in an RFC or part of an RFC would help us maybe not to forget about it. But I'm open for all and think it's a good idea, just am not the right one to implement it.

**Comment by richard67** [comment]
One thing we should keep in mind is permissions. A logged in frontend user who has the "Edit own" permission but not the permission to create new articles should not have the "Save as Copy" option.

**Comment by joomdonation** [comment]
I made PR https://github.com/joomla/joomla-cms/pull/32520 to implement this PR for this PR. Please test !

**Comment by ghost** [comment]
We see no need in this feature. Is it possible to turn off this feature?

**Reply by richard67** [reply]
@chnnst Who is „we“? Do you think you speak for the Community? Or do you want to pretend you do, and that‘s why you have changed your user name here to one which looks pretty much like the one of a maintainer?

**Reply by ghost** [reply]
Who is „maintainer“? Who is richard67? Do you think you speak for the Community? The question is Is it possible with an option button to turn this feature off? @richard67 Do you think you can discriminate on name basis? Or do you want to pretend you can use awkward way to ask about name but be silent on if Is it possible with an option button to turn this feature off?

This new feature creates new issues, please stop proposing them to j4 but better propose them to j4.1 or later .

We want Joomla 4 to release NOW, so please stop wasting time of other people, and better test release blockers.

**Comment by chmst** [comment]
Thank you @joomdonation!
